app/models/asset_field_type.rb

```diff
@@ -34,6 +34,7 @@ def data
         'dimensions': dimensions,
         'content_type': asset_content_type,
         'file_size': asset_file_size,
+        'has_thumb': is_image(asset_content_type),
```

Comment:
Ok but in asset_info that was the preexisting assumption 

```ruby
- if asset
  = image_tag(asset['style_urls'][config[:thumbnail_style]], height: '50px')
```

additionally this is also used in Show where an `image_tag` is rendered as well, I made this function to and image element wont have a docx file as a source